README.md

```diff
@@ -7,3 +7,4 @@ This repo is to say goodbye to one of the greatest pull request reviewers we hav
 
 ## Comments and Wishes
 
+Fare thee well Svet! The ultimate PR ninja. All the best at the new gig, you're going to crush it !important - Paul
```

Comment:
He stole the "PR Ninja" from me...where's my credit Paul?
Classic dev who steals other people's code and submits as their own..